attractions/autocomplete/autocomplete-option.svelte

```diff
@@ -1,8 +1,19 @@
 <script>
+  /**
+   * @typedef {{name: string; details?: string}} Option
+   * @event {{ nativeEvent: MouseEvent }} click
+   */
   import { createEventDispatcher } from 'svelte';
   import escapeRegExp from '../utils/escape-regexp.js';
 
+  /**
+   * @type {Option} The option data to render
```

Comment:
Is that even valid? Looks rather strange, it'd be better to put the description above the type (and the full stop).

Reply:
yeah, that's just me being mindless